dist/obs-bundled-gems.spec

```diff
@@ -105,7 +105,12 @@ bundle config build.nokogiri --use-system-libraries
 
 bundle --local --path %{buildroot}%_libdir/obs-api/
 
+# Make sure rake and rack in Gemfile.lock match the versions from the
+# rubygem-rack and ruby2.7 packages
+# otherwise Passenger wont start the app
+
 test -f %{buildroot}%_libdir/obs-api/ruby/2.7.0/gems/rack-%{rack_version}/rack.gemspec
+test -f %{buildroot}%_libdir/obs-api/ruby/2.7.0/gems/rack-%{rake_version}/rack.gemspec
```

Comment:
Is this line correct? rack != rake. it looks like you copied the line and changed only rack_version -> rake_version but not changed the gem name.

Reply:
For sure not! Thanks